contents/docs/apps/url-query.md

```diff
@@ -0,0 +1,48 @@
+---
+title: URL query parameter converter
+github: https://github.com/PostHog/integrations-repository/pull/31
+installUrl: https://app.posthog.com/project/apps?name=url-query-parameter-converter
+thumbnail: ../../apps/thumbnails/url-query
+tags:
+    - url-query
+---
+
+### What does the URL query parameter converter app do?
+
+This app automatically converts URL query parameters for specific terms into event properties in PostHog, enabling you to analyse them like any other data point. This can be useful for seeing how your product or content converts based on search terms. 
```

Comment:
@joethreepwood Minor nit, but not sure how this can be true. Search terms aren't carried within URL parameters, only pre-defined UTMs based on campaigns... unless I'm misunderstanding what you mean here?